### DISCUSSION TITLE
ServerSideMilvusException{errorCode=UNEXPECTED_ERROR, reason=Illegal data size for chunk field: objectFeature}

### DISCUSSION THREAD

**Discussion by ranzhinahongchen** [milvus-io/milvus#18697] [Q&A and General discussion]
我在插入数据的时候遇到了一个问题，ServerSideMilvusException{errorCode=UNEXPECTED_ERROR, reason=Illegal data size for chunk field: objectFeature}，请问下这个是因为字段长度超出限制的原因吗，应该怎么解决呢，这个字段的类型是vector_float

**Comment by yhmo** [comment]
Which Milvus version?

**Reply by ranzhinahongchen** [reply]
milvus_cpu_0.11.0

**Reply by yhmo** [reply]
> milvus_cpu_0.11.0

这个版本是实验性质的，当时没做完就撤了，我也只能翻翻代码。
“Illegal data size for chunk field”这个错误发生在Insert()的时候，错误原因估计是客户端给过来的InsertParam的各个field里的数组长度不一致。

**Reply by ranzhinahongchen** [reply]
已经找到了解决方案，dimension要和InsertParam的直接对应，之前以为这个dimension类似mysql的最大长度
